sprocs/group_users_insert.sql

```diff
@@ -11,6 +11,9 @@ DECLARE
     arg_group_id bigint;
     arg_cur_size bigint;
     arg_max_size bigint;
+    arg_has_roles boolean;
+    arg_default_group_role_id bigint;
+    arg_required_roles_count bigint;
```

Comment:
Nit: I know you're just pattern-matching against existing code, but we shouldn't use the `arg_` prefix for things that are not actually arguments. I'd appreciate it if you could update both the new and old variables here.